paddle/framework/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
```

Comment:
Shall we also use `cpplint` as a complementary ?  Because it already in `cmake/clint.cmake`, maybe we can invokes the wrapper function `add_style_check_target` in generic functions.